### PR TITLE
2 theme

### PR DESCRIPTION
**Pull request by CapitalX**
Tweaked the theme to closer match DETA, also tweaked some example syntax based on feedback.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -4,13 +4,12 @@ repo_url: 'https://github.com/detahq/docs'
...
-    accent: 'pink'
+    primary: 'grey'
+    accent: 'purple'
   font:
-    text: Roboto
-    code: Fira Mono
-  logo:
-    icon: 'book'
+    text: Avenir
+    code: Courier
```
Comment (abdelhai) [inline mkdocs.yml]: Courier is ugly. Why change Fira?

Comment (CapitalX) [reply mkdocs.yml]: Courier was more visible on the page than Fira or IBM Plex Mono. They are both dispalying a bit light.

Mkdocs doesn't make it super easy to load in a custom font weight and when I try to increase it for Fira or IBM it becomes way too bold.

Comment (CapitalX) [reply mkdocs.yml]: Switched back to Fira for now.

**Comment by abdelhai** [comment]
Fonts are not being fetched from the server. i get a 403 from google fonts server.

**Pull request merged**